harmony_netcdf_to_zarr/convert.py

```diff
@@ -163,8 +167,14 @@ def __copy_group(src, dst):
     for name, item in src.groups.items():
         __copy_group(item, dst.create_group(name.split('/').pop()))
 
+    procs = []
+    fork_ctx = multiprocessing.get_context('fork')
```

Comment:
I don't really know how this multi-processing fork works, but in general if there are 1000s of variables, is this smart enough to not try and fork 1000 processes? I'm thinking of something like Clojure's functionality which takes the number of cores into account for parallel functions.